data/856/885/49/85688549.geojson

```diff
@@ -688,6 +688,9 @@
     ],
     "wof:breaches":[],
     "wof:concordances":{
+        "digitalenvoy:region_code":[
+            4
```

Comment:
@stepps00 should this have a `04` zero prefix?

Also, looks like this is a list when it should just be single value? (We put lists in for the alt concordances only.)

Reply:
Looks like the original value is simply "4". I'll dig into this a bit more because the wikipedia page you linked to above does show `0` prefix values for some codes.. but I think this is ok as-is.

As for the list vs string values.. I included these as lists because many WOF features map to more than one DigitalEnvoy concordance. Even if I stored a single concordance value in the `wof:concordances` property and the remainders in the `wof:concordances_alt` property, the alt concordances would still need to be a list since there could be one or more alt concordance values.

I don't think there are any existing concordance lists, but I'm not sure if there is another way to store these values.

Reply:
I think the WOF way to do this is single values for the default concordance, and more values go into the list of alt concordances lists. Else the tools that parse default concordances as single values will barf.

Reply:
That is correct.

Reply:
fixed in ed03296